tsconfig.json

```diff
@@ -8,6 +8,7 @@
   "compilerOptions": {
     "esModuleInterop": true,
     "lib": ["ES2017", "DOM"],
+    "strict": true,
```

Comment:
I enabled this to get a failing test on `test_js` after adding the route with default null, otherwise the compiler happily compiled the project